app/services/fetch_link_card_service.rb

```diff
@@ -18,6 +18,8 @@ def call(status)
     return if res.code != 200 || res.mime_type != 'text/html'
 
     attempt_opengraph(card, url) unless attempt_oembed(card, url)
+  rescue HTTP::ConnectionError, OpenSSL::SSL::SSLError
```

Comment:
The exception is raised when the connection fails or the HTTPS connection fails with `http_client.head (url)`.